### PR TITLE
Removed sponsors section

### PR DESCRIPTION
**Pull request by vedasri-g**
I replaced sponsors section in Readme.md with a surprise

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,23 +17,7 @@ Want to create reveal.js presentation in a graphical editor? Try <https://slides
...
-        </div>
-        <b>Your app, enterprise-ready.</b>
-        <div>
-          <sub>Start selling to enterprise customers with just a few lines of code. Add Single Sign-On (and more) in minutes instead of months.</sup>
-        </div>
-      </a>
-    </td>
-  </table>
-</div>
+Veda was HEREEEEE!
```
Comment (VerySpeedyPhoton) [inline README.md:20]: haha cute!

File: README.md [hunk 2]
```diff
@@ -17,23 +17,7 @@ Want to create reveal.js presentation in a graphical editor? Try <https://slides
 
 ---
 
-### Sponsors
```
Comment (VerySpeedyPhoton) [inline README.md:20]: could you leave *just* the sponsors line?

**Review by VerySpeedyPhoton — COMMENTED** [review]
Beautiful! just one adjustment please...

File: README.md (hunk 1 above)
Comment (vedasri-g) [reply README.md:20]: Thank you

Comment (VerySpeedyPhoton) [reply README.md:20]: Thank you for incorporating the change!

**Review by VerySpeedyPhoton — APPROVED** [review]

**Pull request merged**